src/Microsoft.Health.Fhir.TemplateManagement/Models/ImageInfo.cs

```diff
@@ -15,6 +16,9 @@ public class ImageInfo
         private const char ImageTagDelimiter = ':';
         private const char ImageRegistryDelimiter = '/';
 
+        // Reference docker's image name format: https://docs.docker.com/engine/reference/commandline/tag/#extended-description
+        private static readonly Regex _imageNameRegex = new Regex(@"^[a-z0-9]+(([_\.]|_{2}|\-+)[a-z0-9]+)*(\/[a-z0-9]+(([_\.]|_{2}|\-+)[a-z0-9]+)*)*$");
```

Comment:
naming convention for static variable s_imageNameRegex. See [coding conventions](https://github.com/dotnet/corefx/blob/eb883d78defbc7d4cae3b8ebd0fa68852eb583e3/Documentation/coding-guidelines/coding-style.md)